docs/AI-for-security/knowledge-base.asciidoc

```diff
@@ -0,0 +1,138 @@
+[[ai-assistant-knowledge-base]]
+= AI Assistant Knowledge Base
+
+AI Assistant's Knowledge Base feature enables AI Assistant to recall specific documents and other specified information. This information, which can include everything from the location of your datacenters to the latest threat research, provides additional context that can improve the quality of AI Assistant's responses to your queries. This topic describes how to enable and add information to Knowledge Base.
+
+NOTE: When you upgrade from {elastic-sec} version 8.15 to a newer version, information previously stored by AI Assistant will be lost. 
+
+.Requirements
+[sidebar]
+--
+
+* To use Knowledge Base, you need the `Elastic AI Assistant: All` privilege. To edit global Knowledge Base entries (information that will affect the AI Assistant experience for other users in the {kib} space), you need the `Allow Changes to Global Entries` privilege. 
```

Comment:
It seems like there are a number of different requirements for AI Assistant and Knowledge Base, depending on what users want to be able to do. Maybe in a future iteration, we could create a separate "AI Assistant requirements" page for these, similar to what we have for some other features.

Reply:
Good idea, thanks